Maestro/subscriptions.json

```diff
@@ -241,6 +276,38 @@
         }
       }
     },
+    // Update dependencies in CoreFX release/2.0.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/2.0.0/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/coreclr/release/2.0.0/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/master/Latest.txt",
```

Comment:
We shouldn't have the corefx and coreclr `master` files in here.  Only the rel/2.0.0 from above.